clarifai/models/model_serving/models/default_test.py

```diff
@@ -0,0 +1,227 @@
+import dataclasses
+import inspect
+import logging
+import os
+import unittest
+
+import numpy as np
+
+from ..model_config.triton_config import TritonModelConfig
+from .output import (ClassifierOutput, EmbeddingOutput, ImageOutput, MasksOutput, TextOutput,
+                     VisualDetectorOutput)
+
+PREDEFINED_TEXTS = ["Photo of a cat", "A cat is playing around"]
+
+PREDEFINED_IMAGES = [
+    np.zeros((100, 100, 3), dtype='uint8'),  #black
+    np.ones((100, 100, 3), dtype='uint8') * 255,  #white
+    np.random.uniform(0, 255, (100, 100, 3)).astype('uint8')  #noise
+]
+
+
+class DefaultTestInferenceModel(unittest.TestCase):
+  """
+  This file contains 2 test cases of:
+  ...
```

Comment:
did you intentionally write ... here?

Reply:
I intended to write a short description here. I updated the content.